src/World.cpp

```diff
@@ -624,6 +624,12 @@ void cWorld::GenerateRandomSpawn(void)
 
 bool cWorld::CheckPlayerSpawnPoint(int a_PosX, int a_PosY, int a_PosZ)
 {
+	// The bottom layer cannot hold a valid spawn point
+	if (a_PosY <= 0)
+	{
+		return false;
+	}
+	
```

Comment:
We haven't specified a policy for that yet, I personally try to avoid trailing whitespace on empty lines, but don't want to edit it out of the entire code-base, since that would totally destroy `git blame`